int32 num_dimensions
float64[] x_range
float64[] y_range
float64[] z_range
float64[] circle_obstacles
float64[] rectangle_obstacles
float64[] ball_obstacles
float64[] box_obstacles
